.changeset/six-berries-admire.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/design-tokens': major
```

Comment:
@connor-baer how should we proceed with upgrading peer dependencies?

We need to release `@sumup/design-tokens@3.0.0` before we can upgrade it in `@sumup/circuit-ui@3.0.0`, so we can release both at once.

Reply:
Huh, that's a real 🐔 or 🥚 problem. I would assume that `changesets` handles this for us, no?

Reply:
Maybe! Hopefully 🤞 Otherwise we'll handle this manually when time comes to release